src/hyperlight_host/src/hypervisor/hyperv_linux.rs

```diff
@@ -629,9 +636,7 @@ impl Hypervisor for HypervLinuxDriver {
         // Then `cancel_requested` will be set to true again, which will cancel the **next vcpu run**.
         // Additionally signals will be sent to this thread until `running` is set to false.
```

Comment:
Might want to correct these comments w/ changing this to an `AtomicU64`.

Reply:
I think it makes sense to keep, while not technically correct, there's still a concept of `running` being true or false (even if it's a bit instead of a bool now)